crates/bevy_asset/src/io/embedded/mod.rs

```diff
@@ -404,4 +411,13 @@ mod tests {
         // Really, should be "my_crate/src/the/asset.png"
         assert_eq!(asset_path, Path::new("my_crate/the/asset.png"));
     }
+
+    #[test]
+    fn remove_embedded_asset() {
```

Comment:
Could this test be expanded to ensure the asset is now returned as not existing? The test currently only checks that you can call `remove_asset` once, but not that it actually removes anything

Reply:
ok i added asserts for that